UnityProject/Assets/Scripts/Lighting/LightTile.cs

```diff
@@ -48,63 +48,63 @@ public void ChangeBrightness(float brightness)
         private IEnumerator PassTheLight(float _brightness, int range)
         {
             //the range is key 1 = closest
-            Dictionary<int, List<Vector2>> radialDispersion = new Dictionary<int, List<Vector2>>();
+            var radialDispersion = new Dictionary<int, List<Vector2>>();
             //the different ranges for the Dictionary
-            for (int i = 1; i <= range; i++)
+            for (var i = 1; i <= range; i++)
             {
                 //row and column length for the box radial
                 int rangeFinder = i + 1 + i;
                 //To store the current range box radial tile positions
-                List<Vector2> lightTiles = new List<Vector2>();
+                var lightTiles = new List<Vector2>();
 
                 //working left to right, for-loop below iterates through the rows
-                for (int k = 1; k <= rangeFinder; k++)
+                for (var k = 1; k <= rangeFinder; k++)
                 {
                     if (k == 1)
                     {
                         //toprow
                         //Starting at top left
-                        Vector2 tilePos = new Vector2(transform.position.x - i,
+                        var tilePos = new Vector2(transform.position.x - i,
                             transform.position.y + i);
-                        for (int tile = 1; tile <= rangeFinder; tile++)
+                        for (var tile = 1; tile <= rangeFinder; tile++)
                         {
                             if (tile == 1)
                             {
                                 lightTiles.Add(tilePos);
                             }
                             else
                             {
-                                Vector2 nextTile = new Vector2(tilePos.x + tile - 1f, tilePos.y);
+                                var nextTile = new Vector2(tilePos.x + tile - 1f, tilePos.y);
                                 lightTiles.Add(nextTile);
                             }
                         }
                     }
                     else if (k == rangeFinder)
                     {
                         //lastrow
-                        Vector2 tilePos = new Vector2(transform.position.x - i,
+                        var tilePos = new Vector2(transform.position.x - i,
                             transform.position.y - i);
-                        for (int tile = 1; tile <= rangeFinder; tile++)
+                        for (var tile = 1; tile <= rangeFinder; tile++)
                         {
                             if (tile == 1)
                             {
                                 lightTiles.Add(tilePos);
                             }
                             else
                             {
-                                Vector2 nextTile = new Vector2(tilePos.x + tile - 1f, tilePos.y);
+                                var nextTile = new Vector2(tilePos.x + tile - 1f, tilePos.y);
```

Comment:
Don't like Vectors declared implicitly

Reply:
Agreed...

Reply:
That basicly end the discussion though:
Rider does not support differentiating different unity types, so automated cleanup is out of the question.

I'll change the cleanup script to reflect thing.

Thank you @fomalsd  👍

Reply:
So end of story: "Var when evident" except for simple-types and do not auto-correct it on cleanup!